public/js/app/views/BookmarkletView.js

```diff
@@ -6,6 +6,18 @@ define(["app/app",
     templateName: 'bookmarklet',
     template: Ember.Handlebars.compile(tpl),
 
+    didInsertElement: function() {
+      var field = this.$("#bookmarklet-recipients")
+      field.select2({ width: '100%' })
+
+      // When the select2 became multiline it change bookmarklet height, but we can't
+      // handle this via CSS rules, so use JS to increase iframe size accordingly
+      // Only way to interact with the scripts outside the iframe is postMessage
+      field.on('change', function() {
+        window.parent.postMessage(window.document.documentElement.offsetHeight, '*')
```

Comment:
I can check the "sender" origin when receiving messages (https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage), but not sure we need this complication here. At least in this bookmarklet version. Even height will be changed by other script it will be restored to correct one after next dropdown change.

Anyway I'll put this into my todo list for the next bookmarklet version for beta.